csrc/optimization/mark_alias.cpp

```diff
@@ -17,6 +17,13 @@ void MarkAliasPass::runPass(Fusion* fusion) {
   const AliasAnalysisResult alias_analysis = findAliases(fusion);
   for (TensorView* out :
        ir_utils::filterByType<TensorView>(fusion->outputs())) {
+    // Lazy move: we could check compatibility and only give up when
```

Comment:
:laughing:  we are moving a lazy move around

Reply:
Yep, "like a rolling stone" :)